filebeat/prospector/prospector_log.go

```diff
@@ -24,7 +24,7 @@ type Log struct {
 	config     prospectorConfig
 }
 
-// NewLog instantiates a new Log
+// NewProspectorLog instantiates a new Log
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
comment on exported function NewLog should be of the form "NewLog ..."